web-app/src/modules/core/routes/Validators/components/ValidatorsStats.tsx

```diff
@@ -2,17 +2,53 @@ import { FC } from 'react';
 import Money from '../../../../ui/Money';
 import StatsCard from '../../../../ui/StatsCard';
 import { IValidator } from '../../../../interface/Validator.interface';
+import { gql, useQuery } from '@apollo/client';
+
+const GET_VALIDATOR_UTILS = gql`
+  query ValidatorUtils {
+    getValidatorUtils {
+      vouchPrice
+      entryFee
+      clearingBid
+      netReward
+    }
+  }
+`;
+
+/*    thermostatMeasure {
+        nextEpoch
+        amount
+        percentage
+        didIncrease
+      }
+*/
 
 interface ValidatorsStatsProps {
   validators?: IValidator[];
 }
 
 const ValidatorsStats: FC<ValidatorsStatsProps> = ({ validators }) => {
+  const { data, error } = useQuery(GET_VALIDATOR_UTILS, {
+    pollInterval: 30000, // Poll every 30 seconds
+  });
+
+  if (error) {
+    console.log('error', error);
+    return <p>{`Error: ${error.message}`}</p>;
+  }
+
   const validatorSet = validators && validators.filter((it) => it.inSet);
   const eligible = validators && validatorSet && validators.length - validatorSet.length;
   const totalLibra = validators && validators.reduce((acc, it) => acc + Number(it.balance), 0);
   const liquidLibra = validators && validators.reduce((acc, it) => acc + Number(it.unlocked), 0);
 
+  // utils
+  const utils = data ? data.getValidatorUtils : null;
+
+  /*const thermostatTitle = utils
```

Comment:
Is this code intentionally commented out?

Reply:
It was a working progress. But removing from this PR now.